### ISSUE TITLE
on eval mode, ground truth labels are concatenated with image_features!

### ISSUE THREAD

**Issue by Hengameh400** [salaniz/pytorch-gve-lrcn#19]
here [code](https://github.com/salaniz/pytorch-gve-lrcn/blob/d1d9d29a96ce803cfb350ae77597434e67fefb57/train/lrcn_trainer.py#L63) , ground truth labels are passed to the model

could you please correct me if i am wrong?

thanks.

**Comment by katelyn98** [comment]
The ids are only used to generate the results dictionary in eval mode, the ids are not being passed into the model.generate_sentence call. See line [here](https://github.com/salaniz/pytorch-gve-lrcn/blob/d1d9d29a96ce803cfb350ae77597434e67fefb57/train/lrcn_trainer.py#L102) in lcrn_trainer.py

**Comment by katelyn98** [comment]
Sorry, nevermind...you are right. The ground truth labels are appended to the image features and passed into the evaluation....not sure where to go from here then...
